addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -507,12 +507,15 @@ ListRenderer.include({
                 onSuccess: resolve,
                 onFailure: reject,
             });
-        }).then(changedFields => {
+        }).then((saveResults) => {
```

Comment:
changing the API here seems wrong: you keep receiving changedFields in non multi edit scenario, i think

Reply:
Maybe by changing it on both sides then ? I'm having a hard time figuring out how to get the previous multiEdit state without the controller sending back a flag to the renderer.